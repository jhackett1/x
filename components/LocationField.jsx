import React from "react"

const LocationField = () => 
    <>
        <label htmlFor="location">Town or postcode</label>
        <input type="text" name="location" id="location"/>
    </>

export default LocationField