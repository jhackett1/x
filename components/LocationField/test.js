import React from "react"
import renderer from "react-test-renderer"
import LocationField from "./index"

it("renders correctly", () => {
    const tree = renderer
        .create(<LocationField/>)
        .toJSON()
    expect(tree).toMatchSnapshot()
})