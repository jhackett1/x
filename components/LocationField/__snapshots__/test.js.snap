// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa jaOyYS"
>
  <label
    className="sc-bwzfXH fMdlDy"
    htmlFor="location"
  >
    Town or postcode
  </label>
  <input
    className="sc-htpNat egThLt"
    id="location"
    name="location"
    placeholder="Try 'Aylesbury'"
    type="text"
  />
  <img
    alt=""
    className="sc-bxivhb hcizMv"
    src={Object {}}
  />
</div>
`;
